auto-discovery/kubernetes/tests/__snapshot__/auto-discovery_test.yaml.snap

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Are the comments fine in the snapshot files? I'm not to familiar with the helm unit tests.

Reply:
Let's just see if the helm tests work with this, if yes then it's fine, if not then we need to add additional `.license` files.

Reply:
The headers seem to be fine